html/pfappserver/root/graph/dashboard.tt

```diff
@@ -336,9 +338,305 @@ END
                     </div>
                 </div>
             </div>
+            <div class="card">
+                <div class="card-title">
+                    <h2>Successful &amp; Unsuccessful RADIUS Authentications</h2>
+                </div>
+                <div class="card-block">
+                    <div class="row-fluid">
+                        <div class="span[% spanSize %]">
+                            <div data-netdata="statsd_gauge_source.packetfence.authentication.success_last_day"
+                            data-host="/netdata/127.0.0.1"
+                            data-common-max="traffic"
+                            data-common-min="traffic"
+                            data-title="number of successful radius authentications in the last day"
+                            data-chart-library="dygraph"
+                            data-colors="#5D737E #6399B5 #BFE8FC #D8F2FF #FCFEFF"
+                            data-height="200px"
+                            data-after="-86400"
+                            data-decimal-digits="0"></div>
+                        </div>
+                        <div class="span[% spanSize %]">
+                            <div data-netdata="statsd_gauge_source.packetfence.authentication.failed_last_day"
+                            data-host="/netdata/127.0.0.1"
+                            data-common-max="traffic"
+                            data-common-min="traffic"
+                            data-title="number of failed radius authentications in the last day"
+                            data-chart-library="dygraph"
+                            data-colors="#5D737E #6399B5 #BFE8FC #D8F2FF #FCFEFF"
+                            data-height="200px"
+                            data-after="-86400"
+                            data-decimal-digits="0"></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="tab-pane[% IF tab == "dhcp" %] active[% END %]" id="dhcp">
+        [% FOREACH interface IN listen_ints %]
+            <div class="card">
+                <div class="card-title">
+                    <h2>DHCP on [% interface %]</h2>
+                </div>
+                <div class="card-block">
+                    <div class="row-fluid">
+                        <div class="span[% spanSize %]">
```

Comment:
You need to recompute `spanSize` according to the loop size. See the section for authentication sources as an example.